common/tvm_utility/include/tvm_utility/pipeline.hpp

```diff
@@ -294,12 +309,37 @@ class InferenceEngineTVM : public InferenceEngine
     return output_;
   }
 
+  /**
+   * @brief Get version information from the config structure and check if there is a mismatch
+   *        between the supported version(s) and the actual version.
+   * @param[in] version_from Earliest supported model version.
+   * @return The version status.
+   */
+  Version version_check(const std::array<char8_t, 3> & version_from) const
```

Comment:
I would expect `version_from` to be fixed as `1.0.0` or `0.0.0` because any number within the range from `version_from` to `version_up_to` is supported, and maybe have some warnings if the version is low.

Reply:
It would up to the package making use of the model to decide. If a model is known to be broken before a certain version, then the maintainer of the package that uses it can pass that version when calling this function.
Also, there can be API changes. It has been the case when going from TVM 0.7 to 0.8 (although that was before the versioning was introduced to ModelZoo), and it will probably be the case again in the future. So models compiled for a version of TVM that uses an older API will be unsupported.